sympy/utilities/lambdify.py

```diff
@@ -88,22 +90,74 @@
     "Matrix": "array",
     "MutableDenseMatrix": "array",
     "ImmutableMatrix": "array",
+    "ImmutableDenseMatrix": "array"
+
+}
+
+# most of the functions of scipy is similar to numpy. Checked each functions.
```

Comment:
I checked all the functions with both scipy and numpy.These are working for both , so these should be similar and I added some additioal functions for Scipy.